frontend/src/app/modules/common/autocomplete/user-autocompleter.component.ts

```diff
@@ -39,6 +39,8 @@ import {NgSelectComponent} from "@ng-select/ng-select";
 import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
 import {HalResource} from "core-app/modules/hal/resources/hal-resource";
 
+export const usersAutocompleterSelector = 'user-autocompleter';
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/3b1eae5a64159f8a044579a25870c772186dbd3f/frontend/src/app/modules/common/autocomplete/user-autocompleter.component.ts#L43"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>